ui/job-view/App.jsx

```diff
@@ -258,10 +258,12 @@ class App extends React.Component {
   };
 
   fetchDeployedRevision() {
+    this.forThis = null; // Just adding to use this
```

Comment:
After enabling ''''class-methods-use-this': 'off'''',  I couldn't figure out how to use this in some of the methods, so I just created a dummy this in them. What's the correct approach for this?